deployment/applicationset/applicationset-chorus-dev.yaml

```diff
@@ -13,9 +13,32 @@ spec:
       revision: HEAD
       files:
       - path: "chorus-dev/*/config.json"
+  strategy:
+    type: RollingSync
+    rollingSync:
+      steps:
+        - matchExpressions:
+            - key: stepName
+              operator: In
+              values:
+                - infrastructure
+        - matchExpressions:
+            - key: stepName
+              operator: In
+              values:
+                - database
+          maxUpdate: 20% # don't break them all at once.
+        - matchExpressions:
+            - key: stepName
+              operator: In
+              values:
+                - application
+          maxUpdate: 20% # don't break them all at once
   template:
     metadata:
       name: '{{index .path.segments 0}}-{{.path.basename}}'
+      labels:
+        stepName: '{{ if hasKey . "stepName" }}{{ .stepName }}{{ else }}application{{ end }}'
```

Comment:
You're going to add the step names in config.json?

Reply:
https://github.com/CHORUS-TRE/environments/pull/124